CHANGELOG.asciidoc

```diff
@@ -26,6 +26,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 This release also includes changes from <<release-3-6-XXX, 3.6.XXX>>.
 
 * Removed `connectOnStartup` configuration option from gremlin-javascript.
+* Added marker interface `PBiPredicate` for predefined predicates.
```

Comment:
should there be provider Upgrade Documentation for this? there are providers with custom predicates. knowing this change was available and when to use it, might be useful to them?